### PR TITLE
Species-specific suit icons

### PR DESCRIPTION
**Pull request by SonixApache**
This implements a few species-specific icons that we had laying around in the files. Specifically, it allows for all the bay races to wear their custom syndie hardsuits, catbeasts get engineering suit sprites, adds two skrell spacesuits, three unathi suits (two with helmets) and adds a syndie commander suit.

catsuits

I moved all the vox-specific shit and all the plasmeme-specific shit to their own files. Of course I did some testing, but I might've missed some.
*To fix the missing icons, usually you just have to add the proper* `species_fit` *var to the item.*

🆑 
 - rscadd: Implemented some species-specific icons we had laying around. Syndi suits for catbeasts, skrell and lizards, engineering suit for catbeasts, two 'new' skrell spacesuits, three 'new' lizard suits and a syndie commander suit.
- experiment: Vox and Plasmaman icons were shuffled around - if you notice any missing or broken icons, please report them.

### CODE REVIEW

File: __DEFINES/global.dm [hunk 1]
```diff
@@ -7,13 +7,15 @@
 #define DNA_SE_LENGTH 58
 
 #define VOX_SHAPED "Vox","Skeletal Vox"
-
 #define GREY_SHAPED "Grey"
-
+#define LIGGER_SHAPED "Unathi"
+#define SKRELL_SHAPED "Skrell"
+#define CATBEAST_SHAPED "Tajaran"
+#define PLASMEME_SHAPED "Plasmaman"
```
Comment (MadmanMartian) [inline __DEFINES/global.dm]: non-dumb defines, please

Comment (SonixApache) [reply __DEFINES/global.dm]: ok

**Comment by Blithering** [comment]
So do ops get the default suit or the HUMAN suit?

**Comment by SonixApache** [comment]
The default suit. The HUMAN suit is now a COMMANDER suit, and is a seperate item.

**Comment by RealestEstate** [comment]
this makes the balids easier to spot behind hardsweets

**Comment by Blithering** [comment]
Any reason why the leftmost lizard suit has no helmet?

**Comment by SonixApache** [comment]
There wasn't a helmet sprite.

**Pull request merged**